web/themes/custom/pineapple/sass/base/_buttons.scss

```diff
@@ -0,0 +1,29 @@
+.js-form-submit.form-submit {
+
+  &#edit-submit--2 {
```

Comment:
Same as above, create button variants and do not use id's

Reply:
done